### ISSUE TITLE
Issue in 0.2.0

### ISSUE THREAD

**Issue by tomelliot16** [nhurel/terraspec#2]
When running 0.2.0 I hit
```
root@fc047254b97d:/terraspec# ../terraspec-linux-x64 --spec=examples/aws/spec/default
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x135ecc7]

goroutine 7 [running]:
github.com/nhurel/terraspec/lib.laxSchema(0x0, 0xc0004c68d0)
	/go/src/github.com/nhurel/terraspec/lib/spec.go:343 +0x37
github.com/nhurel/terraspec/lib.decodeBody(0xc00007f060, 0xc000040a20, 0x20, 0xc00006b700, 0xc00008b6d0, 0x0, 0x0, 0x0, 0xc00007ee80, 0x0, ...)
	/go/src/github.com/nhurel/terraspec/lib/spec.go:296 +0x134
github.com/nhurel/terraspec/lib.ParseSpec(0xc00015c800, 0x58b, 0x78b, 0xc0000b4750, 0x28, 0xc00006b700, 0x0, 0x0, 0x1b2ea80, 0x0)
	/go/src/github.com/nhurel/terraspec/lib/spec.go:259 +0x231
github.com/nhurel/terraspec/lib.ReadSpec(0xc0000b4750, 0x28, 0xc00006b700, 0xc0000cbe00, 0x1, 0x1, 0x0)
	/go/src/github.com/nhurel/terraspec/lib/spec.go:219 +0x1a4
main.PrepareTestSuite(0x17677c5, 0x1, 0xc00021ddd0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/nhurel/terraspec/main.go:152 +0x2ca
main.runTestCase(0xc00021ddd0, 0xc0000d2540)
	/go/src/github.com/nhurel/terraspec/main.go:92 +0x5d
main.main.func1(0xc0000d2540, 0xc000042a40, 0xc00021ddd0)
	/go/src/github.com/nhurel/terraspec/main.go:64 +0x35
created by main.main
	/go/src/github.com/nhurel/terraspec/main.go:63 +0x1d5
```
when running in 0.1.6 it seemed to work fine however there are a lot of features not in 0.1.6.
```
root@fc047254b97d:/terraspec/examples/aws# /terraspec/terraspec-linux-x64 --spec=spec/default/
🏷  default
 ✔  module.ec2-instance.aws_instance.this[0].associate_public_ip_address = false
 ✔  module.ec2-instance.aws_instance.this[0].credit_specification[0].cpu_credits = standard
 ✔  module.ec2-instance.aws_instance.this[0].disable_api_termination = false
 ✔  module.ec2-instance.aws_instance.this[0].ebs_optimized = false
 ✔  module.ec2-instance.aws_instance.this[0].get_password_data = false
 ✔  module.ec2-instance.aws_instance.this[0].instance_type = t2.small
 ✔  module.ec2-instance.aws_instance.this[0].monitoring = false
 ✔  module.ec2-instance.aws_instance.this[0].source_dest_check = true
 ✔  module.ec2-instance.aws_instance.this[0].subnet_id = mocked_subnet_id
 ✔  module.ec2-instance.aws_instance.this[0].tags.Name = my-instance
 ✔  module.ec2-instance.aws_instance.this[0].tenancy = default
 ✔  module.ec2-instance.aws_instance.this[0].volume_tags.Name = my-instance
```

**Comment by nhurel** [comment]
Hello,
I've just published release 0.3. 
Can you tell me if you still have the issue ?

**Comment by tomelliot16** [comment]
WORKS!!!!!!
